doc/src/conf.py

```diff
@@ -32,8 +32,10 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'breathe', 'sphinx.ext.mathjax', 'sphinx.ext.viewcode', 'sphinx_rtd_theme'
+    'breathe', 'sphinx.ext.mathjax', 'sphinx.ext.viewcode', 'sphinx_rtd_theme',
+    'sphinx.ext.autosectionlabel'
```

Comment:
Just curious, what does the extension provide?

Reply:
It enables automatically  labelling / creating reference to other sections via 
```
:ref:`dev/tools:rocTX`
```
Found it here: https://stackoverflow.com/a/40225340